oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSourceDBRecord.java

```diff
@@ -160,8 +160,8 @@ protected void writeNonNullToDB(PreparedStatement stmt, Schema fieldSchema,
         String timestampString = Timestamp.valueOf(localDateTime).toString();
         Object timestampWithTimeZone = createOracleTimestamp(stmt.getConnection(), timestampString);
         stmt.setObject(sqlIndex, timestampWithTimeZone);
-      } else if (Schema.LogicalType.TIMESTAMP_MICROS.equals(fieldSchema.getLogicalType())) {
-        // Deprecated: Handle the case when the Timestamp is mapped to CDAP Timestamp type
+      } else {
```

Comment:
How are we ensuring that this condition will be executed in case of `Timestamp` or `Date` type?

Reply:
In prepareRun and configurePipeline class, we have validateSchema calls which will validate it.

Reply:
It should be `DateTime` according to https://github.com/data-integrations/database-plugins/blob/f0347b9572c7c4a1032a8e64186119d5adf1f448/oracle-plugin/src/main/java/io/cdap/plugin/oracle/OracleSource.java#L189

Reply:
Ah I see, makes sense.